Makefile

```diff
@@ -323,3 +323,12 @@ frontend-i18n-generate: \
 frontend-i18n-compile: ## Format the crowin json files used deploy to the apps
 	cd $(PATH_FRONT) && yarn i18n:deploy
 .PHONY: frontend-i18n-compile
+
+# -- K8S
+build-k8s-cluster: ## build the kubernetes cluster using kind
+	./bin/start-kind.sh
+.PHONY: build-k8s-cluster
```

Comment:
Just a small note: Is `build` the most accurate verb to use here? Does the shell script actually build something, or would a different term be more precise?

Reply:
[fixup! 🔨(CI) add Tilt](https://github.com/numerique-gouv/people/pull/178/commits/77bf4271f39fcc03fc1f2bacc0812126772c2e2d)